CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contributing Guidelines
+
+## Commit messages
+Each PR should consist of a single commit with the following format:
+```
+Commit title no longer than 80 chars
+
+Commit message can be as long as you want. It needs to be complete but minimal,
+meaning it should contain all the information to understand the change but
+exclude redundant information. Please keep the lines in the commit message up
+to 80 chars as well.
+```
+
+The reason for requesting to squash you PR into a single **logical** commit is because our merge policy will squash all the commits anyway before merging. When squashed by Github it will list all the commit titles and messages as bullets, therefore, it will be more informative if correctly written to begin with.
```

Comment:
```suggestion
The reason for requesting to squash your PR into a single **logical** commit is because our merge policy will squash all the commits anyway before merging. When squashed by Github it will list all the commit titles and messages as bullets, therefore, it will be more informative if correctly written to begin with.
```